microcosm_pubsub/dispatcher.py

```diff
@@ -19,7 +19,8 @@
 @logger
 @defaults(
     # Number of failed attempts after which the message stops being processed
-    message_max_processing_attempts=typed(int, default_value=None)
+    message_max_processing_attempts=typed(int, default_value=None),
```

Comment:
What does this do? Does this behave differently than SQS's `maxReceiveCount` config?

Reply:
That's an old thing that was put in for local/integration testing. In those cases we use LocalStack, which doesn't deadletter. Instead we keep a count of the number of times the message was processed. It's unused by default though.